wsd/DocumentBroker.hpp

```diff
@@ -1460,7 +1460,7 @@ class DocumentBroker : public std::enable_shared_from_this<DocumentBroker>
     const bool _alwaysSaveOnExit;
 
     // Last member.
-#ifdef ENABLE_DEBUG
```

Comment:
Note: #ifdef ENABLE_DEBUG is always true as config.in will always define it. In a production build config.in contains the following:

#define ENABLE_DEBUG 0

Since _unitWsd was always instantiated in Online builds, production or not, I removed the #ifdef ENABLE_DEBUG.

Reply:
Shouldn't we fix `ENABLE_DEBUG` instead? (Test cannot run in non-debug builds.)

I'd rather we keep this and fix configure.ac. Technically, it's correct that we need test code in debug builds only.